CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v.6.0.0
+ - **BREAKING**: methods `deleteRepository`, `isCollaborator`, `addCollaborator`, `removeCollaborator`, `testPushHook`, `pingHook`, and `deleteHook` now return a `Future<void>` instead of a `Future<bool>`.
+ - **BREAKING**: parameters of method `createDeployKey` have been modified.
+ - **BREAKING**: parameter `targetCommitsh` in `Release` class has been renamed as `targetCommitish`
```

Comment:
If you end up adding the deprecated method and make this non-breaking, this can be downgraded to a non-breaking change.